cmake/Modules/FindCppCMS.cmake

```diff
@@ -0,0 +1,14 @@
+if (CPPCMS_INCLUDE_DIR AND CPPCMS_LIBRARY)
+
+  # in cache already
+  SET(CppCMS_FOUND TRUE)
```

Comment:
This looks weird, did you try to find a FindCppCMS.cmake in other projects? Maybe a robust solution exists already out there.